### ISSUE TITLE
Can you define an "empty data" representation?

### ISSUE THREAD

**Issue by holmescharles** [johnkerl/miller#1030]
I have a pprint formatted file where I represent NULL or EMPTY data with the string "-99999". In python if I want to treat that string as null while I read the file, I use the command `pandas.read_csv(FILENAME, na_values=-99999)`.
Is there a way to do this with miller?

**Comment by johnkerl** [comment]
For `pprint` I use `_`:

```
$ cat foo.csv
a,b,c
1,2,3
4,,6

$ mlr --icsv --opprint cat foo.csv
a b c
1 2 3
4 - 6
```

However (old, old, _old_ bug/feature): the PPRINT reader isn't doing the reverse:

```
$ cat foo.tbl
a b c
1 2 3
4 - 6

$ mlr --ipprint --ocsv cat foo.tbl
a,b,c
1,2,3
4,-,6
```

**Comment by johnkerl** [comment]
Note pprint format is the only file format that needs this -- for all other formats, empty string is fine.

**Comment by johnkerl** [comment]
As a workaround:

```
mlr --ipprint --ocsv --from foo.tbl \
  put '
    for (k,v in $*) {
      if (v == "-") {
        $[k] = "";
      }
    }
  '
```

(This is clumsy)

**Comment by johnkerl** [comment]
Fixed in #1059
